pkg/apis/extensions/types.go

```diff
@@ -288,9 +278,8 @@ type RollbackConfig struct {
 const (
 	// DefaultDeploymentUniqueLabelKey is the default key of the selector that is added
 	// to existing RCs (and label key that is added to its pods) to prevent the existing RCs
-	// to select new pods (and old pods being select by new RC). See DeploymentSpec's UniqueLabelKey
-	// field for more information.
-	DefaultDeploymentUniqueLabelKey string = "deployment.kubernetes.io/podTemplateHash"
+	// to select new pods (and old pods being select by new RC).
+	DefaultDeploymentUniqueLabelKey string = "pod-template-hash"
```

Comment:
The desire for lack of a namespace prefix was one reason I had originally insisted on the key being configurable -- so that the user could change the key in order to avoid key-name collisions with their own labels.

However, in the short term it's easier if we don't support that field. If users request it, we can add it back later.